pkg/workflows/workflows_integration_test.go

```diff
@@ -30,7 +30,7 @@ func TestIntegrationCreateWorkflow(t *testing.T) {
 	notificationTriggers := []AiWorkflowsNotificationTrigger{"ACTIVATED"}
 
 	// Create a workflow to work with in this test
-	workflowInput := generateCreateWorkflowInput(channel, notificationTriggers)
+	workflowInput := generateCreateWorkflowInput(channel, notificationTriggers, nil)
```

Comment:
Sadly there's no real way to check the flag as it's only relevant for SLACK.
For the moment I'm just checking the fact that it's nil, but in the future we will expand its usage further so more tests can be done.